File: 0287-find-the-duplicate-number/0287-find-the-duplicate-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findDuplicate = function(nums) {
    let ans = 0;
    nums.sort();
    for(let i = 1; i < nums.length; i++)
        {
            if(nums[i] === nums[i - 1])
                {
                    ans = nums[i];
                }
        }
    return ans;
};

File: 0287-find-the-duplicate-number/0287-find-the-duplicate-number.cs
public class Solution {
    public int FindDuplicate(int[] nums) {
        int ans = -1;
        Array.Sort(nums);

        for(var i = 0; i < nums.Length -1; i++)
        {
            if(nums[i] == nums[i+ 1])
            {
                ans = nums[i];
            }
        }
        return ans;
    }
}